libs/libdstdec/dst_init.c

```diff
@@ -80,17 +80,27 @@ RT:  Richard Theelen, PDSL-labs Eindhoven <r.h.m.theelen@philips.com>
 static void *MemoryAllocate(int NrOfElements, int SizeOfElement) 
 {
   void *Array;
-
+#ifdef __arm__
+  if ((Array = malloc(NrOfElements * SizeOfElement)) == NULL)
+  {
+    fprintf(stderr,"ERROR: not enough memory available!\n\n");
+  }
+#else
   if ((Array = _mm_malloc(NrOfElements * SizeOfElement, 16)) == NULL) 
   {
     fprintf(stderr,"ERROR: not enough memory available!\n\n");
   }
+#endif
   return Array;
 }
 
 static void MemoryFree(void *Array) 
 {
+#ifdef __arm__
```

Comment:
This should be 
`#if defined(__arm__) || defined(__aarch64__)`
to cover 64-bit ARM devices.